src/librustdoc/html/static/js/main.js

```diff
@@ -57,7 +58,7 @@ function resourcePath(basename, extension) {
     return getVar("root-path") + basename + getVar("resource-suffix") + extension;
 }
 
-(function () {
+((() => {
```

Comment:
Is it possible to not do it in this case? It's an anonymous function, not a callback after all. ^^'